frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.ts

```diff
@@ -675,6 +675,8 @@ export class ExperimentDesignComponent implements OnInit, OnChanges, OnDestroy {
         (decisionPoint) => decisionPoint.target === aliasRowData.target && decisionPoint.site === aliasRowData.site
       );
 
+      console.log({ decisionPoint });
```

Comment:
did you forgot to remove this console.log or you leave it for any reason?

Reply:
oops, nope, ty needs removed